File: src/main/scala/resonantinduction/compatibility/nei/RICrusherRecipeHandler.java
package resonantinduction.compatibility.nei;

import resonant.lib.utility.LanguageUtility;
import resonantinduction.core.RecipeType;

public class RICrusherRecipeHandler extends RITemplateRecipeHandler
{

	@Override
	public String getRecipeName()
	{
		return LanguageUtility.getLocal("resonantinduction.machine.crusher");
	}

	@Override
	public RecipeType getMachine()
	{
		return RecipeType.CRUSHER;
	}
}

File: src/main/scala/resonantinduction/compatibility/nei/RICrusherRecipeHandler.scala
package resonantinduction.compatibility.nei

import resonant.lib.utility.LanguageUtility
import resonant.lib.factory.resources.RecipeType

class RICrusherRecipeHandler extends RITemplateRecipeHandler
{
    def getRecipeName: String =
    {
        return LanguageUtility.getLocal("resonantinduction.machine.crusher")
    }

    def getMachine: RecipeType =
    {
        return RecipeType.CRUSHER
    }
}